web/gds-user-ui/src/constants/basic-details.ts

```diff
@@ -68,3 +68,27 @@ export const getBusinessCategoryOptions = () => {
 export const getBusinessCategiryLabel = (category: string) => {
   return vaspCategories.find((c) => c.value === category)?.label;
 };
+
+export const getBusinessCategoryValue = (category: string) => {
+  switch (category) {
+    case 'UNKNOWN_ENTITY':
+      return BUSINESS_CATEGORY.UNKNOWN_ENTITY;
+    case 'PRIVATE_ORGANIZATION':
+      return BUSINESS_CATEGORY.PRIVATE_ORGANIZATION;
+    case 'GOVERNMENT_ENTITY':
+      return BUSINESS_CATEGORY.GOVERNMENT_ENTITY;
+    case 'BUSINESS_ENTITY':
+      return BUSINESS_CATEGORY.BUSINESS_ENTITY;
+    case 'NON_COMMERCIAL_ENTITY':
+      return BUSINESS_CATEGORY.NON_COMMERCIAL_ENTITY;
+    default:
+      return category;
+  }
+};
```

Comment:
I'm not sure if this is necessary. Instead, you can retrieve the value using `BUSINESS_CATEGORY[data]`. This function appears to have redundant functionality. It would be appreciated if it could be removed.

Reply:
This has been removed after sorting out the error I was getting below.